googledatastore/conf/googledatastore.properties

```diff
@@ -0,0 +1,92 @@
+# Copyright (c) 2015 YCSB contributors. All rights reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"); you
+# may not use this file except in compliance with the License. You
+# may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or
+# implied. See the License for the specific language governing
+# permissions and limitations under the License. See accompanying
+# LICENSE file.
+
+#
+# Sample property file for Google Cloud Datastore DB client 
+
+## Mandatory parameters
+# See this link for instructions about setting up Google Cloud Datastore and
+# authentication:
+# https://cloud.google.com/datastore/docs/getstarted/start_java/
+#
+# After you setup your environment, you will have 3 pieces of information ready:
+# datasetId, service account email, and a private key file. These must be
+# configured via the properties below
+#
+# googledatastore.datasetId=<string id of your dataset>
+# googledatastore.privateKeyFile=<full path to your private key file>
+# googledatastore.serviceAccountEmail=<Your service account email>
```

Comment:
GoogleDatastoreClient.java is actually expecting ```googledatastore.serviceAccount``` as the property name.

Reply:
What exactly is the googledatastore.datasetId=<string id of your dataset> referring to? I read on another forum that it is Project ID but when I update the properties file with the Project ID the following error is returned:

site.ycsb.DBException: Required property "datasetId" missing.
        at site.ycsb.db.GoogleDatastoreClient.init(GoogleDatastoreClient.java:113)
        at site.ycsb.DBWrapper.init(DBWrapper.java:86)
        at site.ycsb.ClientThread.run(ClientThread.java:91)
        at java.base/java.lang.Thread.run(Thread.java:832)

I've tried using the Project Number but to no avail.